site/content/en/docs/contribution guidelines/testing.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Running tests in Kueue"
```

Comment:
I would suggest moving this page out of Contribution Guidelines to "Tasks -> Developer Tools -> Running tests". I think we can safely skip "Kueue" since this is the name of the project.